src/library_pthread.js

```diff
@@ -426,14 +407,14 @@ var LibraryPThread = {
             createScriptURL: (ignored) => new URL("{{{ TARGET_JS_NAME }}}", import.meta.url)
           }
         );
-        worker = new Worker(p.createScriptURL('ignored'), workerOptions);
+        worker = new Worker(p.createScriptURL('ignored'), {{{ WORKER_OPTIONS }}});
```

Comment:
I simpler way to do this would be to define a new macro here at the top of this file. e.g.

```
{{{
globalThis.WORKER_OPTIONS = {
   ...
};
null;
}}}
```

Reply:
Thank you so much! I wasn't too sure how the templating works in these files 😅 . I'll update the PR with this approach when I get a chance.